### PR TITLE
Feat:Nightly build is failure for rd-commondata-dataload

### PR DESCRIPTION
**Pull request by shivajihmcts**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RDCC-5235

### Change description ###
Nightly build is failure for rd-commondata-dataload

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java [hunk 1]
```diff
@@ -162,4 +165,70 @@ private List<Categories> getLovServicesCase2() {
                 .build()
         );
     }
+
+    private List<Categories> getLovServicesCase3() {
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java:169]: Is this being used?

File: src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java [hunk 2]
```diff
@@ -162,4 +165,70 @@ private List<Categories> getLovServicesCase2() {
...
+                .key("BBA3-001AD")
+                .valueEN("ADVANCE PAYMENT new")
+                .parentCategory("caseType")
+                .parentKey("BBA3-001")
+                .active("N")
+                .build()
+        );
+    }
+
+    private List<Categories> getLovServicesCase4() {
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java:202]: Is this being used?

File: build.gradle [hunk 3]
```diff
@@ -119,13 +119,13 @@ jacocoTestReport {
 pitest {
   junit5PluginVersion = '0.15'
   targetClasses = ['uk.gov.hmcts.reform.rd.commondata.*']
-  testSourceSets = [sourceSets.test]
+  testSourceSets = [sourceSets.test, sourceSets.functionalTest]
```
Comment (AdilOoze) [inline build.gradle:122]: How long will Mutation tests take to run now that you have added Functional Tests also?

Comment (shivajihmcts) [reply build.gradle:122]: Around 12 mins on my local

File: src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java (hunk 1 above)
Comment (shivajihmcts) [reply src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java]: Removed

File: src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java (hunk 2 above)
Comment (shivajihmcts) [reply src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java]: Removed

File: src/functionalTest/resources/sourceFiles/categories/list_of_values_duplicate_rec_case5.csv [hunk 4]
```diff
@@ -0,0 +1,5 @@
+CategoryKey,ServiceID,Key,Value_EN,Value_CY,Hinttext_EN,Hinttext_CY,Lov_Order,ParentCategory,ParentKey,Active
+caseSubType,BBA3,BBA3-001AD,ADVANCE PAYMENT new,,,,,caseType,BBA3-001,
+caseSubType,BBA3,BBA3-001AD,ADVANCE PAYMENT scenario4,,,,,caseType,BBA3-001,Y
+caseSubType,BBA4,BBA3-001AD,ADVANCE PAYMENT scenario4,,,,,caseType,BBA3-001,N
+caseSubType,BBA3,BBA3-001AD,ADVANCE PAYMENT new,,,,,caseType,BBA3-001,M
```
Comment (AdilOoze) [inline src/functionalTest/resources/sourceFiles/categories/list_of_values_duplicate_rec_case5.csv:5]: Is this needed? I can't see any test case referencing it?

File: src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java [hunk 5]
```diff
@@ -162,4 +165,5 @@ private List<Categories> getLovServicesCase2() {
                 .build()
         );
     }
+
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java:168]: Please rollback this file if there are no changes

File: src/functionalTest/java/uk/gov/hmcts/reform/rd/commondata/cameltest/CommonDataCategoriesLoadTest.java [hunk 6]
```diff
@@ -298,6 +298,7 @@ void testListOfValuesCsv_DupRecord_Case4() throws Exception {
         );
     }
 
+
```
Comment (AdilOoze) [inline src/functionalTest/java/uk/gov/hmcts/reform/rd/commondata/cameltest/CommonDataCategoriesLoadTest.java:301]: Please rollback this file if there are no changes

File: src/functionalTest/resources/sourceFiles/categories/list_of_values_duplicate_rec_case5.csv (hunk 4 above)
Comment (shivajihmcts) [reply src/functionalTest/resources/sourceFiles/categories/list_of_values_duplicate_rec_case5.csv]: Removed

File: src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java (hunk 5 above)
Comment (shivajihmcts) [reply src/test/java/uk/gov/hmcts/reform/rd/commondata/camel/processor/CategoriesProcessorTest.java]: done

File: src/functionalTest/java/uk/gov/hmcts/reform/rd/commondata/cameltest/CommonDataCategoriesLoadTest.java (hunk 6 above)
Comment (shivajihmcts) [reply src/functionalTest/java/uk/gov/hmcts/reform/rd/commondata/cameltest/CommonDataCategoriesLoadTest.java:301]: done

**Review by AdilOoze — APPROVED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

**Pull request merged**